app/api/resources/mentorship_relation.py

```diff
@@ -100,8 +102,14 @@ def put(cls, request_id):
         user_id = get_jwt_identity()
         response = DAO.accept_request(user_id=user_id, request_id=request_id)
 
-        return response
+        if response[1] is 200:
+            request = MentorshipRelationModel.find_by_id(request_id)
+            sender_id = request.action_user_id
```

Comment:
Same here, there is no use of sender_id, it acts as intermediate, so you can also combine these two lines into one.
sender = UserModel.find_by_id(request.action_user_id). But this is a minor tradeoff between clean code and complex code. So, what would you suggest about this?